### ISSUE TITLE
Error when uninstalling multiple packages in one go

### ISSUE THREAD

**Issue by gokulsoumya** [pipxproject/pipx#309]
**Describe the bug**
An unrecognized argument error is thrown when multiple package names are passed to the uninstall command.

**How to reproduce**
```
pipx uninstall beets asciinema
usage: pipx [-h] [--version]
            {install,inject,upgrade,upgrade-all,uninstall,uninstall-all,reinstall-all,list,run,runpip,ensurepath,completions}
            ...
pipx: error: unrecognized arguments: asciinema
```

**Expected behavior**
Uninstall all packages mentioned.

**Comment by cs01** [comment]
This is how the API is designed: one package can be uninstalled at a time. We could change it so that it accepts multiple packages, but as it is designed, this is expected behavior.

**Comment by gokulsoumya** [comment]
Ah so it's not a bug. But it would be intuitive and more useful if it could accept multiple packages.

**Comment by clbarnes** [comment]
> Ah so it's not a bug. But it would be intuitive and more useful if it could accept multiple packages.

The same is true for installation, but that's much harder, for other reasons. I would prefer uninstallation to reflect installation that for one to support multiple packages and the other not.

**Comment by cs01** [comment]
I agree that it's nice to keep install/uninstall symmetric. I am not sure I see why it's much harder to install multiple packages in one go though.

**Comment by clbarnes** [comment]
I suppose I was thinking about the --spec option, which loses meaning a bit when there's more than one package to install, but I suppose we could disallow multiple arguments when --spec is used.

**Comment by itsayellow** [comment]
There's no longer a `--spec` option for `install` (#302) but there are other less-used per-package options.
